allofplos/article_class.py

```diff
@@ -943,11 +972,16 @@ def remote_tree(self):
     @property
     def journal(self):
         """Journal that an article was published in.
-
         Can be PLOS Biology, Medicine, Neglected Tropical Diseases, Pathogens,
         Genetics, Computational Biology, ONE, or the now defunct Clinical Trials.
+        Relies on a simple doi_to_journal transform when possible, and uses `self.get_plos_journal()`
```

Comment:
Now all of the journal logic is jumping out at me as being fairly complicated, could that also be moved into the elements submodule? Or is there a reason not to do that?